plugin/models.go

```diff
@@ -62,8 +62,7 @@ func FindPluginByID(ctx context.Context, id string) (*Plugin, error) {
 		return nil, err
 	}
 
-
-	res, err := utils.GetMongoDBDoc(PluginCollectionName, bson.M{"_id": objID, "deleted": false})
+	res, err := utils.GetMongoDBDoc(PluginCollectionName, bson.M{"_id": objID})
```

Comment:
Why is the 'deleted' variable removed?

Reply:
The plugin struct has no **deleted** field, the filter query never matches, when I created a sample plugin to test the fix, returns not found when you find by id but it'll be there when you list all plugins.

Reply:
Please revert as the change is not the priority of your fix and I am not aware of any issue as regards that section of the code. I do not see why you need to adjust that without prior verification of why it is done so.

Reply:
Ok, if you say so.

Reply:
done